src/lib/es.js

```diff
@@ -123,7 +131,9 @@ function buildQuery(parameters) {
   }
 
   const datetimeQuery = buildDatetimeQuery(parameters)
-  if (datetimeQuery) {
+  if (datetimeQuery instanceof Error) {
+    throw datetimeQuery
+  } else if (datetimeQuery) {
```

Comment:
The `else` isn't needed here. If the condition on line 134 hits, then you just wouldn't get this far.

Reply:
if the `datetime` parameter isn't defined, `datetimeQuery` will be undefined, and we only want to push int into the ES query if it is defined.

Reply:
```suggestion
  }
  
  if (datetimeQuery) {
```